readme.txt

```diff
@@ -338,13 +338,18 @@ Please [contribute your translation](https://github.com/TycheSoftwares/woocommer
 
 == Changelog ==
 
+= 5.0.2 (12.06.2024) =
+* Fix - HTML tags are not functioning properly on the invoice.
+* Fix - Fatal error when placing the order.
```

Comment:
@gaminikhil Please mention that the fatal error was coming due to conflict with packing slips

Reply:
Done